google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/executor/AppEngineExecutor.java

```diff
@@ -42,8 +45,23 @@ public void cancel() {
     }
   }
 
+  @SuppressWarnings("FutureReturnValueIgnored")
   private void setProcess(Process process) {
     this.process = process;
+    ThreadUtil.getInstance()
```

Comment:
feels like this shouldn't live in a `setProcess` method which is just a simple setter. can this be split out and called alongside `setProcess` from the `run` method?